book/src/building-from-source.md

```diff
@@ -162,3 +162,54 @@ file. For example, to use `kitty`:
 sed -i "s|Exec=hx %F|Exec=kitty hx %F|g" ~/.local/share/applications/Helix.desktop
 sed -i "s|Terminal=true|Terminal=false|g" ~/.local/share/applications/Helix.desktop
 ```
+
+### Building the Deb package
+
+#### Provided Deb limitation
+
+The .deb package should be provided on the release page. But because the CI
+build is currently using Ubuntu 22.04 (this can change in the future, find the
+real image version in the .github/workflows/release.yml) the `.deb` has a certain
+libc6 version dependency. You can find it within the specific `.deb` details.
+Currently libc6 (>= 2.34). That is why you can't use this deb on systems older than
+Ubuntu 22.04, Mint 21, or Debian 12.
```

Comment:
This text may age poorly as versions change. Let's say something a bit more generic like "The CI may use a libc version greater than what your Ubuntu/Debian/Mint version requires - in this case you can build a `.deb` from source." and jump right into the directions.